recipes/mingw/all/conandata.yml

```diff
@@ -0,0 +1,18 @@
+sources:
+  "8.1":
```

Comment:
MinGW version and gcc version are two different things. What's the version of MinGW-w64 here?

Reply:
version : MinGW-W64-builds-4.3.5
user    : nixman
date    : 05.12.2018-10:29:36 AM

Reply:
nixman's personal builds are [not maintained any more by the way](https://sourceforge.net/p/mingw-w64/discussion/723797/thread/ea9a5b00fb/). We probably want to use something more future-proof, for example http://winlibs.com/

Reply:
Agree, but it provides only posix/seh for 64bits, and posix/dwark for 32bits

Reply:
Yes, only few configurations available.

Reply:
What about http://mingw-w64.org?

Reply:
@uilianries For me, it's not resolved, the version is misleading. When I see mingw/8 recipe, I don't expect to have MinGW 4.3.5 ;)